src/components/ToDo/styles/LastRow.styled.ts

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components'
+
+export const Rectangle = styled.div<{ isDarkMode: boolean }>`
+  height: 30px;
+  background-color: ${(props) => (props.isDarkMode ? 'hsl(235, 24%, 19%)' : 'hsl(0, 0%, 98%)')};
+  border-radius: 5px;
+  color: ${(props) => (props.isDarkMode ? 'hsl(234, 39%, 85%)' : 'hsl(235, 19%, 35%)')};
+  box-shadow: ${(props) => (props.isDarkMode ? '2px 2px 4px #000000' : '')};
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 15px 25px 15px;
+`
+export const Circle = styled.button`
+  height: 25px;
+  width: 25px;
+  background-color: transparent;
+  border-radius: 50%;
+  border: 2px solid hsl(234, 39%, 85%);
+  cursor: pointer;
+`
+export const ItemsLeft = styled.div`
+  font: Josefin Sans;
```

Comment:
should this be font-family?

Reply:
actually i think `font:` is a shorthand
<img width="1222" alt="Screenshot 2021-12-05 at 4 58 07 AM" src="https://user-images.githubusercontent.com/57870728/144724392-773d41a9-93ab-4e1d-b31d-745e272c9af1.png">

but in that case, font-weight and font-size should also be under this line